single_group_playbooks/repo.yml

```diff
@@ -19,6 +19,6 @@
     - basic_security
     - remove
     - update
-    - pulp_server
-    - {role: logs_client, when: stacks_logs_servers is defined}
+#    - pulp_server
+#    - {role: logs_client, when: stacks_logs_servers is defined}
```

Comment:
Is there a reason why `logs_client` role is commented out for the `repo` servers?

Reply:
No, that was a left-over mistake from debugging. Obviously the `pulp_server` also should not be disabled on repo servers. Will correct this in the PR for part 4.